public/emails/ng_alert_notification.html

```diff
@@ -1,10 +1,8 @@
 <!doctype html>
-<html xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
+<html lang="und" dir="auto" xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
 
 <head>
-  <title>
-    {{ Subject .Subject .TemplateData "{{ .Title }}" }}
-  </title>
+  <title>{{ Subject .Subject .TemplateData "{{ .Title }}" }}</title>
```

Comment:
Would it be possible to remove formatting changes like this, once we pull in changes from upstream, this would probably cause a code conflict.